ci/py36-env.yml

```diff
@@ -0,0 +1,23 @@
+name: matplotcheck-dev
```

Comment:
@nkorinek this needs to be earthpy-dev as you called it above on line 40 

`activate-environment: earthpy-dev`

you'll need to change all three envt files for this to work